### PR TITLE
EVG-7735 resize volumes

### PR DESCRIPTION
**Pull request by ablack12**

### CODE REVIEW

**Comment by ybrill** [comment]
did you decide not to automatically [expand the partition/filesystem](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html)?

**Comment by tychoish** [comment]
(yeah, I think filesystem resizing is worth splitting out to separate/future work.)

File: cloud/ec2_client.go [hunk 1]
```diff
@@ -595,6 +598,34 @@ func (c *awsClientImpl) DeleteVolume(ctx context.Context, input *ec2.DeleteVolum
 	return output, nil
 }
 
+// ModifyVolume is a wrapper for ec2.ModifyWrapper.
+func (c *awsClientImpl) ModifyVolume(ctx context.Context, input *ec2.ModifyVolumeInput) (*ec2.ModifyVolumeOutput, error) {
+	var output *ec2.ModifyVolumeOutput
+	var err error
+	msg := makeAWSLogMessage("ModifyVolume", fmt.Sprintf("%T", c), input)
```
Comment (tychoish) [inline cloud/ec2_client.go:605]: can we add the volumeID to this?

Comment (ablack12) [reply cloud/ec2_client.go:605]: The volume ID is part of the input.

**Comment by ablack12** [comment]
@ybrill were you planning to review or should I merge? Either way is fine, just didn't want to cut you off.

**Comment by ybrill** [comment]
thanks! I wasn't planning to review

**Pull request merged**